examples/tweet/content.js

```diff
@@ -35,7 +36,7 @@ export class Content extends React.Component {
   }
   render() {
     let { media, text } = this.props
-    let styles = this.styles 
```

Comment:
Mine does too, I end up using sublime to edit these files, which isn't the greatest :/ I would've suggested using prettier on the entire project but I think the intent is to move glamor's implementation to typescript.